api/src/main/java/org/accula/api/handlers/AdminHandler.java

```diff
@@ -0,0 +1,22 @@
+package org.accula.api.handlers;
+
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+import static org.springframework.web.reactive.function.server.ServerResponse.badRequest;
+import static org.springframework.web.reactive.function.server.ServerResponse.ok;
+
+@Component
+public class AdminHandler {
+    private static final String MESSAGE = "You are the admin now.";
+    private static final String ERROR = "Access denied. Try to login as ADMIN.";
+
+    @NotNull
+    public Mono<ServerResponse> getAdmin() {
+        return ok().bodyValue(MESSAGE)
+                // Unbelievable :)
+                .or(badRequest().bodyValue(ERROR));
```

Comment:
It's an unreachable branch, isn't it?

Reply:
Yes, there is a comment.

Reply:
Comment sounds like "It's hard to believe, but this line is reachable" for me. So it doesn't matter